packages/react/src/ReactSharedInternals.js

```diff
@@ -6,13 +6,52 @@
  */
 
 import assign from 'object-assign';
+import {
+  unstable_cancelScheduledWork as cancelScheduledWork,
+  unstable_now as now,
+  unstable_scheduleWork as scheduleWork,
+} from 'react-scheduler';
+import {
+  __getInteractionsRef,
+  __getSubscriberRef,
+  unstable_clear as clear,
+  unstable_getCurrent as getCurrent,
+  unstable_getThreadID as getThreadID,
+  unstable_track as track,
+  unstable_wrap as wrap,
+} from 'react-scheduler/tracking';
+import {
+  unstable_subscribe as subscribe,
+  unstable_unsubscribe as unsubscribe,
+} from 'react-scheduler/tracking-subscriptions';
 import ReactCurrentOwner from './ReactCurrentOwner';
 import ReactDebugCurrentFrame from './ReactDebugCurrentFrame';
 
 const ReactSharedInternals = {
   ReactCurrentOwner,
   // Used by renderers to avoid bundling object-assign twice in UMD bundles:
   assign,
+  // Re-export the react-scheduler API(s) for UMD bundles.
+  // This avoids introducing a dependency on a new UMD global in a minor update,
+  // Since that would be a breaking change (e.g. for all existing CodeSandboxes).
+  Scheduler: {
+    cancelScheduledWork,
+    now,
+    scheduleWork,
+  },
+  SchedulerTracking: {
+    __getInteractionsRef,
+    __getSubscriberRef,
+    clear,
+    getCurrent,
+    getThreadID,
+    track,
+    wrap,
+  },
+  SchedulerTrackingSubscriptions: {
+    subscribe,
+    unsubscribe,
+  },
```

Comment:
These will hang from `React` secret exports in CJS too even though CJS `ReactDOM` doesn't use it, is that correct?

In that case I think maybe it's time to fork this file into the UMD and normal versions. Or 
make a `__UMD__` constant (might be simpler). It would be unfortunate to have all these names being unminifiable for an object that ends up being unused.

Reply:
Yes that's true. I considered forking but it felt gross. Maybe a __UMD__ constant would be less gross. 😄 Lemme look into that.

Reply:
Not sure if this is correct but might be the reason.

<img width="374" alt="screen shot 2018-08-31 at 7 04 48 pm" src="https://user-images.githubusercontent.com/810438/44928755-c215d600-ad50-11e8-9b39-875c666f9a1b.png">

Reply:
The PR blurb mentions the reason I believe we're seeing this:
> As part of this change, the react-scheduler code (which used to be inlined into individual renderers) has been moved to the react package. This makes it look like react has grown more than it has (rather, there is a corresponding decrease in e.g. react-dom).

Reply:
But this would only impact the UMD bundle ^

Reply:
Aaah I forgot it's measuring UMD